in-depth/submission_guidelines.md

```diff
@@ -86,6 +86,10 @@ Heading
 
 : The name of your theme or plugin.
 
+View
+
+: Choose between the image oriented `jumbovision` layout, or the text-oriented `card` layout.  The default for Themes is `jumbovision`, while plugins use `card`.  This option only needs to be specified if you'd like to use the opposite layout.
```

Comment:
Using a different layout leads in 99% of the cases to rejecting the submission (I even can't think of a single reason why a plugin/theme should use a non-default layout, this parameter actually is for things like `zz_contribute.md` only), so I don't think it's necessary to mention `view` at all.